Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -11331,3 +11347,11 @@ private function __ScanFileMTime pFilename, pRecursive, pAllowMissing, \
       throw "Failed to check modification time: missing file"
    end if
 end __ScanFileMTime
+
+function revIDEUTF8FileContents pFile
+   return textDecode(url("binfile:" & pFile), "utf-8")
```

Comment:
This might need to look for and strip a BOM, since `textDecode()` doesn't currently take care of that.